repair/row_level.cc

```diff
@@ -462,10 +468,13 @@ class repair_writer {
     future<> write_start_and_mf(lw_shared_ptr<const decorated_key_with_hash> dk, mutation_fragment mf, unsigned node_idx)  {
         _current_dk_written_to_sstable[node_idx] = dk;
         if (mf.is_partition_start()) {
-            return _mq[node_idx]->push_eventually(mutation_fragment_opt(std::move(mf)));
+            return _mq[node_idx]->push_eventually(mutation_fragment_opt(std::move(mf))).then([this, node_idx] {
+                _stats[node_idx].nr_parttion_start++;
+            });
```

Comment:
Is this racy? Suppose we start to push and block, then someone stops the repair. We see unincremented nr_parttion_start.

Reply:
We will wait for the push to complete before we check the abort flag.

Reply:
Where?

Reply:
See ri.check_in_abort() in the code.